Gems/GradientSignal/Code/Source/Components/GradientTransformComponent.cpp

```diff
@@ -83,7 +84,9 @@ namespace GradientSignal
                     ->Attribute(AZ::Edit::Attributes::ReadOnly, &GradientTransformConfig::IsAdvancedModeReadOnly)
                     ->DataElement(0, &GradientTransformConfig::m_bounds, "Bounds", "Local (untransformed) bounds of a box used to remap, clamp, wrap, scale incoming coordinates")
                     ->Attribute(AZ::Edit::Attributes::ReadOnly, &GradientTransformConfig::IsBoundsReadOnly)
-                    
+                    ->DataElement(0, &GradientTransformConfig::m_bounds, "Center", "Local (untransformed) center of a box used to remap, clamp, wrap, scale incoming coordinates")
```

Comment:
If this stays here, it should use m_center, not m_bounds.

Reply:
good catch, thanks